ui/tests/integration/two-step-button-test.js

```diff
@@ -135,4 +137,68 @@ module('Integration | Component | two step button', function(hooks) {
       );
     });
   });
+
+  test('when in the prompt state, clicking outside will reset state back to idle', async function(assert) {
+    const props = commonProperties();
+    this.setProperties(props);
+    await render(commonTemplate);
+
+    click('[data-test-idle-button]');
```

Comment:
This is an existing test so maybe it’s just about following what’s already here, but I was wondering if it wouldn’t make sense to use the page object component here?

Reply:
Ah yeah I think this makes sense! I wrote this test before I made the TwoStepButton page object. I'll go back and update it.